app/models/review.rb

```diff
@@ -84,16 +84,12 @@ def fields_for_heading(heading)
     end
   end
 
-  def has_scale(heading)
-    scale_field(heading).present?
+  def prepare_heading_html_attribute(heading)
+    heading.downcase.gsub(/\s/,'-')
```

Comment:
Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.